bundles/net.tourbook.cloud/src/net/tourbook/cloud/messages.properties

```diff
@@ -1,103 +1,89 @@
-Log_DownloadWorkoutsToSuunto_003_AllWorkoutsAlreadyExist = 0 workouts to be downloaded: All the workouts found in Suunto App already exist in the database
-Log_DownloadWorkoutsToSuunto_004_NoSpecifiedFolder       = A destination folder need be specified to download the new workouts from Suunto App
-Log_DownloadWorkoutsToSuunto_005_DownloadStatus          = {0} -> Workout Downloaded to the file: "{1}"
-Log_DownloadWorkoutsToSuunto_006_FileAlreadyExists       = {0} -> The following file already exists at the location: "{1}"
-Log_DownloadWorkoutsToSuunto_007_Error                   = The following error occurred when retrieving the workout of Id {0}: {1}
-
-Log_UploadToursToSuunto_001_Start           = Uploading {0} Routes to Suunto App
-Log_UploadToursToSuunto_002_NoGpsCoordinate = {0} -> GPS coordinates are missing
-Log_UploadToursToSuunto_003_UploadStatus    = {0} -> Uploaded Route Id: "{1}"
-Log_UploadToursToSuunto_004_UploadError     = {0} -> Error while uploading the route: "{1}"
-
-Pref_AccountInformation_SuuntoApp_WebPage_Link = https://www.suunto.com/suunto-app/suunto-app/
+Dialog_DropboxBrowser_Button_ParentFolder_Tooltip = Go to the parent folder
+Dialog_DropboxBrowser_Button_SelectFolder         = &Select Folder
+Dialog_DropboxBrowser_Text                        = Choose a Dropbox Folder that will be accessed by MyTourbook to import new tours
+Dialog_DropboxBrowser_Text_AbsolutePath_Tooltip   = Current folder's absolute path
+Dialog_DropboxBrowser_Title                       = Choose a Dropbox Folder
 
-Pref_Checkbox_Use_SinceDateFilter         = &Use a date filter
-Pref_Checkbox_Use_SinceDateFilter_Tooltip = Only workouts for which the start date is greater than this date will be downloaded
+Dialog_UploadRoutesToSuunto_Message = Uploaded tours: {0}\n\
+                                      Tours with errors: {1}
+Dialog_UploadRoutesToSuunto_SubTask = {0} Routes Preparation\n\
+                                      {1} Upload
+Dialog_UploadRoutesToSuunto_Task    = Uploading {0} tours as Suunto Routes
+Dialog_UploadRoutesToSuunto_Title   = Suunto App Routes Upload Summary
 
-Pref_Combo_Workouts_FolderPath_Combo_Tooltip = This is the folder where the new workout files will be downloaded to
-Pref_Combo_Workouts_Label_FolderPath         = Download &folder
+Dialog_UploadToursToStrava_Message = Uploaded tours: {0}\n\
+                                     Tours with errors: {1}
+Dialog_UploadToursToStrava_SubTask = {0} Tour Preparation\n\
+                                     {1} Upload
+Dialog_UploadToursToStrava_Task    = Uploading {0} tours to Strava ({1})
+Dialog_UploadToursToStrava_Title   = Strava Tour Upload Summary
 
-Suunto_Workouts_Description = Download workouts from a Suunto App account
+Dialog_ValidatingSuuntoTokens_SubTask = \u231B Validating the Suunto App tokens
 
-ValidatingSuuntoTokens_SubTask = \u231B Validating the Suunto App tokens
+Html_CloseBrowser_Text = You can now close this browser's window
```

Comment:
`Html_CloseBrowser_Text`

A better name would be `Html_Text_CloseBrowser` for the possible case that additional `Html_Text_...`  text keys are needed

Reply:
Fixed

Reply:
Fixed in this [PR](https://github.com/wolfgang-ch/mytourbook/pull/310)